airflow/providers_manager.py

```diff
@@ -281,7 +281,7 @@ def _add_hook(self, hook_class_name: str, provider_package: str) -> None:
                     self._add_customized_fields(provider_package, hook_class, field_behaviours)
 
         except Exception as e:  # noqa pylint: disable=broad-except
-            if os.environ.get("AIRFLOW_INSTALLATION_METHOD") != ".":
+            if os.environ.get("AIRFLOW_INSTALLATION_METHOD") != "." and not isinstance(e, ImportError):
```

Comment:
Could you update the commend in the else block to refer to this case/behaviour please?

Reply:
Yep. Good point.